tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+
+
+*** Test Cases ***
+Verify Pytorch And Tensorflow Can Be Spawned
+    [Documentation]    Verify Pytorch and tensorflow can be spawned
+    [Tags]    Sanity
+    ...       RHODS-1469
+    Verify Cuda Builds Are Completed
+    Verify Tensorflow Image Can Be Spawned
+    Verify Pytorch Image Can Be Spawned
+
```

Comment:
I would add a Test Teardown to close the jupyter server

Reply:
Added End Web Test in "Verify Image Can Be Spawned" for closing server